handbook/engineering/hiring/engineering-manager-backend-platform.md

```diff
@@ -0,0 +1,72 @@
+- You are motivated to work on ambitious long term goals to solve complex technical challenges.
+- You can effectively reason about and communicate the value of platform projects to stakeholders.
+- You thrive in making plans with your team, and communicate those appropriately.
+- You are [high agency](https://www.linkedin.com/pulse/high-agency-its-importance-how-cultivate-shreyas-doshi/).
+
+## Your responsibilities
+
+- You will facilitate a cohesive team environment where everyone is happy, motivated, and productive.
+- You will ensure that the team has [well defined goals that are aligned with the business](https://about.sourcegraph.com/handbook/engineering/cloud#goals) that are effectively communicated to all levels of the company, and that the team is making continuous iterative progress towards those goals every week.
+- You will help engineers grow in their careers.
+- You will hire exceptional engineers to grow the team in a sustainable way so that the team can accomplish more over time.
+
+## Qualifications
+
+- You have 2+ years of experience directly managing at least 6 full-time software engineers. We expect you already know how to lead a team.
+- You have done some amount of coding in the past 2 years (for example: at work or on a personal project) to keep your skills up-to-date. We expect that you are able to review and provide valuable feedback on code, architecture, and designs.
+- You are able to clearly communicate your past accomplishments verbally (example: in an interview) and in writing (examples: in your resume, in your answers to our application questions).
+- You have experience breaking up complex technical problems into smaller achievable milestones that the team can iterate on.
+
+## Nice to haves
+
+- You have experience designing, architecting, scaling, and optimizing systems that span multiple layers of a tech stack (example: frontend, backend, CLI).
+- You have grown at least one junior engineer to a senior+ level.
+- You have designed a software engineer interview process from first principles.
+- You have built and delivered an application that works on multiple platforms (examples: different operating systems, cloud versus on-premise)
+
+## Learn more about us
+
+To create a product that serves the needs of all developers, we are building a diverse [all-remote team](../../../company/remote/index.md) that is [distributed across the world](../../../company/team/index.md). Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+We provide [competitive compensation](../../people-ops/compensation.md) and [practical benefits](../../people-ops/benefits-and-perks.md) to keep you happy and healthy so that you can do your best work.
+
+Learn more about what it is like to work at Sourcegraph by reading [our handbook](../../index.md)
+
+## Interview process
+
+1. You apply here (TODO - create Lever job)
```

Comment:
```suggestion
<!-- 1. You [apply here](#todo). --><!-- This is commented out because I will be sending some outbound messages before accepting applications. -->
```